### PR TITLE
Issue/234 cant save image

### PR DESCRIPTION
**Pull request by vallo87**
fix in checkRequestForUpload of ModulesComponent

### CODE REVIEW

File: tests/TestCase/Controller/Component/ModulesComponentTest.php [hunk 1]
```diff
@@ -155,6 +156,101 @@ public function testGetProject($expected, $meta) : void
...
+
+    /**
+     * Test `isAbstract()` method.
+     *
+     * @dataProvider isAbstractProvider();
+     * @covers ::isAbstract()
+     *
+     * @return void
+     */
+    public function testIsAbstract($expected, $data) : void
```
Comment (stefanorosanelli) [inline tests/TestCase/Controller/Component/ModulesComponentTest.php]: Add tests and coverage on `ModulesComponent` is a good idea, but better do it in a different PR
These methods are not related to the current issue

Comment (vallo87) [reply tests/TestCase/Controller/Component/ModulesComponentTest.php]: Ok, I will separate it

File: src/Template/Pages/Model/view.twig [hunk 2]
```diff
@@ -15,6 +15,7 @@
         {{ Form.create({'defaults': resource.attributes, 'schema': {}}, {
             'url': {'_name': 'model:save', 'resource_type': resourceType},
             'id': 'form-main',
+            '@submit': 'handleSubmit',
```
[the remark this answers is not in the document]
Comment (vallo87) [reply src/Template/Pages/Model/view.twig]: My bad, forgot to remove it

**Comment by vallo87** [comment]
while I tested it I didnt recon it wasn't working properly
will fix
